### DISCUSSION TITLE
新版本桌面客户端有延迟测试的方法吗？

### DISCUSSION THREAD

**Discussion by MWT6433** [natfrp/wiki#453] [使用问题]
更新桌面客户端后延迟测试功能被取消了，这是为了减轻服务器负担吗？请问有其他方法可以实现延迟测试吗？

**Comment by MWT6433** [comment]
或者说能对丢包率进行测试也可以，有一些需要高可靠性的场景

**Comment by fakeboboliu** [comment]
取消延迟测试是因为他被广泛误解并误用了。

类似功能可能还会有，
但是我们暂时还没能找到一个适合 在目前网络条件下 通过几秒钟的测试 体现一整天网络链路特征和稳定性的指标。

需要更高可靠性的话请开启多个隧道，请不要误解 ping 的本意。
